### PR TITLE
Fixes to compile ros indigo on Gentoo.

### PR DESCRIPTION
**Pull request by bluecube**
Requires overlays ros and Gentoo-ROS-overlay (https://github.com/steup/Gentoo-ROS-Overlay)

### CODE REVIEW

File: rosdep/base.yaml [hunk 1]
```diff
@@ -692,6 +692,7 @@ gazebo:
     raring: [gazebo]
     saucy: [gazebo2]
     trusty: [gazebo2]
+  gentoo: [sci-electronics/gazebo]
```
Comment (wjwwood) [inline rosdep/base.yaml]: Any idea what version of `gazebo` this is?

File: rosdep/base.yaml [hunk 2]
```diff
@@ -2172,6 +2177,7 @@ liburdfdom-headers-dev:
     trusty: [liburdfdom-headers-dev]
     utopic: [liburdfdom-headers-dev]
     vivid: [liburdfdom-headers-dev]
+  gentoo: [media-libs/urdfdom_headers]
```
Comment (wjwwood) [inline rosdep/base.yaml]: Where these (`console_bridge`, `urdfdom`, and `urdfdom_headers`) contributed by you or someone else into portage? Any idea what versions?

**Comment by wjwwood** [comment]
Our CI enforces alphabetical ordering: https://travis-ci.org/ros/rosdistro/builds/58676112#L192

File: rosdep/base.yaml (hunk 2 above)
Comment (bluecube) [inline rosdep/base.yaml]: These are from the Gentoo-ROS-Overlay I linked in the description. Versions 0.2.7, 0.3.0 and 0.3.0.

File: rosdep/base.yaml (hunk 1 above)
Comment (bluecube) [inline rosdep/base.yaml]: Gazebo comes from the main gentoo repository and is version 5.1.0. Is this ok?

**Comment by bluecube** [comment]
I was a little too fast with making this pull request. Other than the ordering there is a one more problem with dependencies between urdfdom and urdfdom_headers... but I'll be back :-)

Btw I have a few questions:
1) What should I do with this pull request?
  - close it and open a new fixed one?
  - add new commits with the fixes?
  - rebase content of the first commit?
2) Can I run the checks that travis does locally?
3) At three places (libvtk-java, python-cairo and python-vtk) I'm using `use:` as in ros-infrastructure/rosdep#176, although It probably doesn't do anything. The problem is that there are no separate packages for language support of libraries and AFAIK the only way to get this is with changed use flags. Is it ok to keep them in the yaml files?

**Comment by wjwwood** [comment]
> 1) What should I do with this pull request?

I would close this, then when you're ready you can reopen it and update the branch it points to with the correct content.

> 2) Can I run the checks that travis does locally?

Yes, just do `nosetest -s` in the root of the repository.

> 3) At three places (libvtk-java, python-cairo and python-vtk) I'm using use: as in ros-infrastructure/rosdep#176, although It probably doesn't do anything. The problem is that there are no separate packages for language support of libraries and AFAIK the only way to get this is with changed use flags. Is it ok to keep them in the yaml files?

Yes, if you plan to eventually update `rosdep` to support them.

**Pull request closed without merging**